modules/jwt_auth_refresh/src/JwtRefreshTokensInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Drupal\jwt_auth_refresh;
+
+use Drupal\Core\Session\AccountInterface;
+
+interface JwtRefreshTokensInterface {
```

Comment:
I really did mean ContentEntityStorageBase. It seems this is specific to loading token entities. A nice pattern might be:

```
$tokens = Drupal::entityTypeManager()->getStorage('jwt_refresh_token')->loadByUid($account->id());
```

This would be supported by extending ContentEntityStorageBase.